LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/MessageProcessor.cs

```diff
@@ -605,8 +605,7 @@ bool IsValidNetId(byte devAddrNwkid, uint netId)
         {
             var netIdBytes = BitConverter.GetBytes(netId);
             devAddrNwkid = (byte)(devAddrNwkid >> 1);
-            byte netIdToCompare = (byte)(netIdBytes[0] & 0b01111111);
-            if (devAddrNwkid.Equals(netIdToCompare))
+            if (devAddrNwkid.Equals(netIdBytes[0] & 0b01111111))
```

Comment:
I meant you can make it return directly
`return (devAddrNwkid.Equals(netIdBytes[0] & 0b01111111))`

instead of 

```c#
if (devAddrNwkid.Equals(netIdBytes[0] & 0b01111111))
  return true;
else
  return false;
```

Reply:
Very true, sorry got somehow over it :/